src/engraving/rw/scorereader.cpp

```diff
@@ -143,6 +144,8 @@ mu::Ret ScoreReader::loadMscz(MasterScore* masterScore, const MscReader& mscRead
         }
     }
 
+    audioSettings= std::move(masterScoreCtx.audioSettingCompat());
```

Comment:
It might be better to make `masterScoreCtx`  as a in-out parameter and set it outside. This will allow us to do other things in the same way, and now if we also need to read something else, we will either have to add another meter or redo this solution...

Reply:
This is just the current implementation, we can change it, for example, replace it with the use the shared pointer

Reply:
AudioSettingsCompat is an out parameter of loadMscz function. To add other things is simple. Extending AudioSettingsCompat is needed only. For example volume and pan of instrument are easily can be added. I think AudioSettingsCompat is readable parametr of loadMscz function instead of ReadContext. ReadContext looks like a useful staff of XmlReader.  Adding it to  loadMscz function seems make this function less understood I guess.

Reply:
fixed